docs/task-files.md

```diff
@@ -328,6 +328,7 @@ The `update-stacks` task will provision all resources in all accounts specified
 | OrganizationBindings             | Dictionary of String, [OrganizationBinding](https://github.com/org-formation/org-formation-cli/blob/master/docs/cloudformation-resources.md#organizationbinding-where-to-create-which-resource) | Set of named OrganizationBindings that can be `!Ref`'d by Resources.<br/><br/> **note**: This value overrides values within the template or resources (value in taskfile is leading).                                                                                               |
 | CloudFormationRoleName           | string                                                                                                                                                                                          | Specifies the name of the IAM Role that must be used to pass to the CloudFormation service. A role with this is expected to exist in the target account (and have the right AssumeRole permissions).                                                                                |
 | TaskRoleName                     | string                                                                                                                                                                                          | Specifies the name of the IAM Role that must be used for cross account access. A role with this is expected to exist in the target account (and have the right AssumeRole permissions).                                                                                             |
+| TaskViaRoleArn                   | string                                                                                                                                                                                          | Specifies ARN of the IAM Role that will be assumed before assuming role specified by the `TaskRoleName`. This role is expected to exist in the target account (and have the right AssumeRole permissions).                                                                                            |
```

Comment:
```suggestion
| TaskViaRoleArn                   | string                                                                                                                                                                                          | Specifies ARN of the IAM Role that will be assumed before assuming role specified by the `TaskRoleName`. This role is expected to have the right AssumeRole permissions, but does not have to exist in the target account |
```

Reply:
thanks! the `TaskViaRoleArn` however does not have to exist in the target account. added a suggestion

Reply:
When you say it does not have to exist, do you mean ofn will create it if it does not exist? In other words, what happens if `TaskViaRoleArn` is specified, but role with the specified ARN does not exist?

Reply:
right, what i meant: does not have to exist in the target account. can exist in a different account, but must exist.
maybe its better to just drop that bit. i updated the suggestion. thanks!

Reply:
I agree, it is best to remove that wording.